utils/wesnoth-map-diff/src/images.ts

```diff
@@ -71,10 +77,12 @@ const produceImagesGetter = async (): Promise<ImagesGetter> => {
         throw new Error(`Missing image for "${baseCode}"`)
       }
 
-      // todo: we should use the defaultBase correctly
+      // TODO: we should use the defaultBase correctly
       if (miscCode) {
         const miscImage = images[`^${miscCode}`]
         if (!miscImage) {
+          // FIXME: this is reached by certain invisible overlays,
+          // such as "^Xo":
```

Comment:
Since this PR is already an attempt to fix this issue, we don't need to include this comment
```suggestion
```